examples/knn/test_knn.cu

```diff
@@ -169,7 +167,7 @@ int main(int argc, char** argv) {
 
   app::Switch_Types<app::VERTEXT_U32B | app::VERTEXT_U64B |
                            app::SIZET_U32B | app::SIZET_U64B |
-                           app::VALUET_F32B | app::UNDIRECTED>(
+                           app::VALUET_F64B | app::UNDIRECTED>(
```

Comment:
Is it possible to enable both F32B and F64B now? Because we may wanna run `float` and `double` for future. As FAISS' KNN is only `float` (32-bit support).